packages/sources/react/src/components/structure/VtmnList/VtmnList.tsx

```diff
@@ -60,12 +89,33 @@ export interface VtmnListItemProps
    * @type {number}
    */
   tabIndex?: number;
+
+  /**
+   * Redirection link
+   * @type {string}
+   */
+  href?: string;
+
+  /**
+   * Target of the link
+   * @type {string}
+   */
+  target?: string;
+
+  /**
+   * Rel of the link
+   * @type {string}
+   */
+  rel?: string;
```

Comment:
Same here maybe we can use `computeRel` to be sure to have the same behavior no matter the language

Reply:
Indeed +1